### ISSUE TITLE
Just basic setup... coding required?

### ISSUE THREAD

**Issue by pindelicato1** [voceconnect/multi-post-thumbnails#142]
Maybe I'm in over my head... does this plugin required edits to php files or other coding?  I was under the impression that I just activate it and the new image input appears in the admin-create post UI... is that not the case?  

Under your quick start wiki topic it appears to suggest that coding is required....

**Comment by rickcurran** [comment]
Yep, it does require a bit of coding to use it in your theme, the Quick Start Wiki topic has the basic instructions.
